maintainers/maintainer-list.nix

```diff
@@ -3816,6 +3816,13 @@
     githubId = 98980;
     name = "Chmouel Boudjnah";
   };
+  ChocolateLoverRaj = {
+    email = "paranjperajas@gmail.com";
+    github = "ChocolateLoverRaj";
+    githubId = 52586855;
+    matrix = "@chocolateloverraj:matrix.org";
+    name = "Rajas Paranjpe";
+  };
```

Comment:
Please put this into a separate commit before your main commit with the commit message `maintainers: add ChocolateLoverRaj`